### PR TITLE
Review Contact Me Form feature-pull request#3

### PR DESCRIPTION
**Pull request by efmomoh**
This feature of the project is the final version of the mobile view portfolio project which was a collaboration of two other team members whose contributions are impressive and immense, thanks to all collaborators for making this feature a success.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,193 +1,377 @@
...
+            placeholder="Write your email"
+            required
+          />
+          <textarea
+            name="textarea"
+            class="text-area"
+            id="text"
+            placeholder="Write your message here..."
+            cols="30"
+            rows="10" required
```
Comment (brainconnect93) [inline index.html]: - [ ] I would kindly suggest you add a `maxlength` attribute to the text-area message and give it a value of `500` as this is part of the project requirement.

File: style.css [hunk 2]
```diff
@@ -621,3 +621,93 @@ button:hover,
...
+/* contact form  */
+.form-container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-image: url("./portfolio-icons/Contact\ me.jpg");
+  background-repeat: no-repeat;
+  height: 779px;
+  margin-top: 30px;
+}
```
Comment (brainconnect93) [inline style.css]: - [ ] I notice you have you background image imported here in your form container, but the image is not showing form my end. i think the the way you have named the image would only make the image to show from your end only and once it gets to the other side the link will get broken. I would kindly advice you rename you image to something like this. `Contact-me.jpg` and target the name using background image property like this `background-image: url("./portfolio-icons/Contact-me.jpg");`. I believe with this the background image will work

| Your Design 👇  | Figma Design 👇 |
| ------------- | ------------- |
| hans | temp |

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @efmomoh, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). ✔️
- Input fields have the `required` attribute to activate the validation ✔️
- Max-length attribute Used on the input ✔️
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (efmomoh) [reply style.css]: When I download the backgroud image the icons in them are not reflecting. what can I do? please help!

Comment (brainconnect93) [reply style.css]: @efmomoh you need to check which image you are downloading from Figma before you export it. You can reach out to me via zoom and let me help you out

File: index.html [hunk 3]
```diff
@@ -1,193 +1,378 @@
...
+            placeholder="Write your email"
+            required
+          />
+          <textarea
+            name="textarea"
+            class="text-area"
+            id="text"
+            placeholder="Write your message here..."
+            cols="30"
+            rows="10" maxlength="500px" required
```
Comment (LionRouge1) [inline index.html]: - [ ] Nice work making the changes :wink: But as requested by the previous reviewer, I suggest you add a `maxlength="500"` (not `maxlength="500px"`) attribute to the text-area message and give it a value of `500` (not `500px`) as this is part of the project requirement.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @efmomoh, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost

## Suggested changes :recycle: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 3 above)
Comment (efmomoh) [reply index.html]: @LionRouge1 kindly take a look again. I have modified it to 500

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @efmomoh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- All requested changes implemented :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**